server/logout/logout_test.go

```diff
@@ -0,0 +1,259 @@
+package logout
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"regexp"
+	"testing"
+
+	"github.com/argoproj/argo-cd/common"
```

Comment:
Nitpick: please remove empty lines between imports.

Reply:
I keep getting this CI lint checking failure ``` File is not `goimports`-ed (goimports)```  when I don't leave spaces, which seems to be resolved by putting spaces between imports